deel-3-webpagina/styles/styles.css

```diff
@@ -1 +1,71 @@
-/*Schrijf hier jouw css*/
\ No newline at end of file
+/*Schrijf hier jouw css*/
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+    background-color: #F2F3D9;
+    font-family: 'Poppins', sans-serif;
+}
+
+nav {
+    display: flex;
+    flex-direction: row;
+    background-color: #151E3F;
+    padding: 40px;
+    justify-content: space-between;
+}
+
+.navUL {
```

Comment:
Bij de uitwerkingen staat hier nog een flex-basis bij, deze: 
flex: 0 1 400px; 

(ik weet niet waarom dat nou echt nodig is)